src/08-tasks/tasks.apib

```diff
@@ -31,6 +31,14 @@ Get a list of tasks.
                 + completed: false (boolean)
                 + completed_at: `2016-02-04T16:44:33+00:00` (string)
                 + due_at: `2016-02-04T16:44:33+00:00` (string)
+                + estimated_duration (object, optional)
+                    + unit: `s` (enum[string], required)
+                        + Members
+                            + s - Seconds
+                    + value: `60` (number, required)
+                + work_type (object, nullable)
```

Comment:
Why is this one `nullable` and the other one `optional`? I guess it should both be `nullable`? Since we always pass the same fields, no matter what in read endpoints?

Reply:
Because i copied it from a Request and this is the reponse :p